pkg/controller/podgc/config/types.go

```diff
@@ -22,4 +22,7 @@ type PodGCControllerConfiguration struct {
 	// before the terminated pod garbage collector starts deleting terminated pods.
 	// If <= 0, the terminated pod garbage collector is disabled.
 	TerminatedPodGCThreshold int32
+	// When set to 'true', Terminated PodGC Threshold will be ignored
```

Comment:
```suggestion
	// When set to 'true', TerminatedPodGCThreshold will be ignored
```

Reply:
i have modified those.

> /remove-sig api-machinery
> 
> Mechanically this looks fine (a few nits commented) -- but before I approve, can you get someone from sig apps to confirm they want this feature, or at least that they don't not want it? :)

cc @foxish would you like to review this PR？thanks